test/clients/rtm/client.js

```diff
@@ -140,6 +140,27 @@ describe('RTM API Client', function () {
 
   });
 
+  describe('Message Updating', function () {
+
+    it('should call a cb with an err when the RTM is not connected', function (done) {
+      var rtm = new RtmAPIClient('fake-token');
+      rtm.updateMessage({}, function (err, res) {
+        expect(err).to.not.equal(null);
+        expect(res).to.equal(null);
+        done();
+      });
+    });
+
+    it('should call a catch cb with an err when the RTM client is not connected', function (done) {
+      var rtm = new RtmAPIClient('fake-token');
```

Comment:
Unfortunately, at this time, it is not. The only way to test this functionality is with a live team. I agree it's far from ideal :( But this might be a good feature to consider for the platform to build!